cpp/arcticdb/version/test/test_version_map_batch.cpp

```diff
@@ -244,9 +244,9 @@ TEST_F(VersionMapBatchStore, CombinedQueries) {
         for(uint64_t j = 0; j < num_versions_per_stream; j++){
             uint64_t idx = i * num_versions_per_stream + j;
             stream_ids.emplace_back(stream);
-            version_queries.emplace_back(VersionQuery{SpecificVersionQuery{static_cast<SignedVersionId>(j)}, false});
+            version_queries.emplace_back(VersionQuery{SpecificVersionQuery{static_cast<SignedVersionId>(j), false}, false});
             stream_ids.emplace_back(stream);
-            version_queries.emplace_back(VersionQuery{TimestampVersionQuery{timestamp(versions[idx]->creation_ts())}, false});
+            version_queries.emplace_back(VersionQuery{TimestampVersionQuery{timestamp(versions[idx]->creation_ts()), false}, false});
```

Comment:
Not for now, but I was thinking we should use enums instead of bools for these. We don't have quite as many as we used to, but do_something(symbol, true, false, false, true, false) is not terribly meaningful and also is not type safe. Something we can maybe refactor towards gradually.